internal/handler/wallet/exportAccount_test.go

```diff
@@ -0,0 +1,60 @@
+package wallet
+
+import (
+	"fmt"
+	"net/http"
+	"net/http/httptest"
+	"testing"
+
+	"github.com/massalabs/thyra-plugin-wallet/api/server/restapi/operations"
+	walletapp "github.com/massalabs/thyra-plugin-wallet/pkg/app"
+	"github.com/massalabs/thyra-plugin-wallet/pkg/wallet"
+	"github.com/stretchr/testify/assert"
+)
+
+func exportWallet(t *testing.T, api *operations.MassaWalletAPI, nickname string) *httptest.ResponseRecorder {
+	handler, exist := api.HandlerFor("GET", "/api/accounts/{nickname}/export")
+	assert.True(t, exist, "Endpoint doesn't exist")
+
+	resp, err := handleHTTPRequest(handler, "GET", fmt.Sprintf("/api/accounts/%s/export", nickname), "")
+	assert.NoError(t, err)
+	return resp
+}
+
+func Test_walletExportAccount_Handle(t *testing.T) {
+	api, prompterApp, resChan, err := MockAPI()
+	assert.NoError(t, err)
+
+	nickname := "walletToExport"
+	password := "zePassword"
+	_, err = wallet.Generate(nickname, password)
+	assert.NoError(t, err)
+
+	t.Run("invalid nickname", func(t *testing.T) {
+		resp := exportWallet(t, api, "toto")
+		verifyStatusCode(t, resp, http.StatusNotFound)
+	})
+
+	t.Run("canceled by user", func(t *testing.T) {
+		resp := exportWallet(t, api, nickname)
+
+		verifyStatusCode(t, resp, http.StatusNoContent)
+
+		prompterApp.App().CtrlChan <- walletapp.Cancel
+
+		_, err = wallet.Load(nickname)
+		assert.NoError(t, err)
```

Comment:
this check is useless. it was used in delete tests to check taht the wallet deletion did not happen.
I think you can remove this whole test case (canceled by user)